pkg/app/app.go

```diff
@@ -85,6 +86,34 @@ func initConfiguration(config configuration.Configuration, apiClient api.ApiClie
 		return orgId
 	})
 
+	config.AddDefaultValue(configuration.ORGANIZATION_SLUG, func(existingValue any) any {
+		client := networking.NewNetworkAccess(config).GetHttpClient()
+		url := config.GetString(configuration.API_URL)
+		apiClient.Init(url, client)
+		if existingValue != nil && len(existingValue.(string)) > 0 {
```

Comment:
Since this value will itself never be set by a user, I think we need to change the logic here a bit. The value specified by the user will be org, holding either the slugname or id of an org.